ci-operator/step-registry/aws/deprovision/iam-role/aws-deprovision-iam-role-commands.sh

```diff
@@ -9,6 +9,10 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 
 REGION="${LEASED_RESOURCE}"
+INFRA_ID=$(jq -r '.infraID' ${SHARED_DIR}/metadata.json)
+CONFIG=${SHARED_DIR}/install-config.yaml
+
+trap 'delete_all' EXIT TERM INT
```

Comment:
Maybe we should move the `trap` line prior to line 12, so that always ensure the trap is run.